usfm_tools/support/singlehtmlRenderer.py

```diff
@@ -262,7 +262,7 @@ def renderFP(self, token):
         self.write(u'<br />')
 
     def renderQSS(self, token):
-        self.write(u'<i>')
+        self.write(u'<i style="float:right;">')
```

Comment:
@richmahn - will this work for RTL languages?

Reply:
@PhotoNomad0 Good question. You know more than me. Does floating right on a page where language is going right to left float left?

Reply:
I should at least class this <i> so that one could override this style with a style sheet.

Reply:
I was just trying to get it to be like this: https://ubsicap.github.io/usfm/poetry/index.html#qs-qs